lib/barbeque/runner.rb

```diff
@@ -0,0 +1,51 @@
+require 'barbeque/config'
+require 'barbeque/exception_handler'
+require 'barbeque/execution_log'
+require 'barbeque/message_handler'
+require 'barbeque/message_queue'
+
+module Barbeque
+  class Runner
```

Comment:
Now it has both "Runner" and "Executor", and they are similar names and it's hard to distinguish their roles from just their names. Could you add comments for their difference to each class?